test/module/shared_model/validators/proposal_validator_test.cpp

```diff
@@ -17,12 +19,47 @@ using namespace shared_model::validation;
 
 class ProposalValidatorTest : public ValidatorsTest {
  public:
-  ProposalValidatorTest() : validator_(iroha::test::kTestsValidatorsConfig) {}
+  ProposalValidatorTest()
+      : validator_(iroha::test::kProposalTestsValidatorsConfig) {}
```

Comment:
Do all the tests require duplicate transactions? Maybe use `kTestsValidatorsConfig` by default, and use `kProposalTestsValidatorsConfig` only in `ProposalWithDuplicateTransactions` test case? Also please add test case when the validation fails.

Reply:
Fixed